sway/input/input-manager.c

```diff
@@ -493,6 +493,14 @@ struct sway_input_manager *input_manager_create(struct sway_server *server) {
 	return input;
 }
 
+void input_manager_finish(struct sway_server *server) {
```

Comment:
Nit: this could take a `struct sway_input_manager *` directly.

Reply:
makes sense

Reply:
done in 86aecdb3.